src/api/app/models/bs_request.rb

```diff
@@ -7,9 +7,9 @@
 class BsRequest < ActiveRecord::Base
 
   class InvalidStateError < APIException
-    setup 'request_not_modifiable', 404
+        setup 'request_not_modifiable', 404
```

Comment:
Use 2 (not 6) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.